packages/strapi-plugin-upload/config/functions/bootstrap.js

```diff
@@ -31,11 +28,35 @@ module.exports = async () => {
   registerPermissionActions();
 };
 
+const wrapFunctionForErrors = fn => async (...args) => {
+  try {
+    return await fn(...args);
+  } catch (err) {
+    if (isError(err, errors.entityTooLarge)) {
```

Comment:
I think you should move that conversion logic in the errors module or at least outside of the wrapFunction util